src/main/java/org/gitlab4j/api/CommitsApi.java

```diff
@@ -1,21 +1,18 @@
 package org.gitlab4j.api;
 
+import org.gitlab4j.api.models.*;
```

Comment:
It is good practice to not use wildcard imports, please change this to only import the types being used.